java/src/main/java/ai/rapids/cudf/RegexFlag.java

```diff
@@ -28,7 +28,8 @@ public enum RegexFlag {
   DEFAULT(0),   // default
   MULTILINE(8), // the '^' and '$' honor new-line characters
   DOTALL(16),   // the '.' matching includes new-line characters
-  ASCII(256);   // use only ASCII when matching built-in character classes
+  ASCII(256),   // use only ASCII when matching built-in character classes
+  EXT_NEWLINE(512); // new-line matches extended characters
```

Comment:
A short blurb here, indicating the definition of the behaviour with `extended characters` would be useful here, for the uninitiated.

The blurb should probably include the fact that when `EXT_NEWLINE` is used, the following characters would be treated as equivalent to a newline:
1. `NEW_LINE('\n')`
2. `CARRIAGE_RETURN('\r)`
3. ...

Reply:
Done!